docs/docs/axes/radial/linear.md

```diff
@@ -133,7 +133,7 @@ The following options are used to configure the point labels that are shown on t
 | ---- | ---- | ------- | ------- | -----------
 | `display` | `boolean` | | `true` | if true, point labels are shown.
 | `callback` | `function` | | | Callback function to transform data labels to point labels. The default implementation simply returns the current string.
-| `font` | `Font` | Yes | `defaults.font` | See [Fonts](../general/fonts.md)
```

Comment:
I'm not sure if this will work out, but we'll see soon :)

Reply:
it worked, check it over here: https://www.chartjs.org/docs/master/axes/radial/linear